lib/order.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'menu'
+
+class Order
```

Comment:
Nice short methods, good. I am a little bewildered that you've declared setters / getters for your instance variables but then accessed those instance variables directly. :-S